specification/marketplaceordering/resource-manager/readme.az.md

```diff
@@ -0,0 +1,112 @@
+## AZ
+
+These settings apply only when `--az` is specified on the command line.
+
+For new Resource Provider. It is highly recommended to onboard Azure CLI extensions. There's no differences in terms of customer usage. 
+
+``` yaml $(az) && $(target-mode) != 'core'
+az:
+    extensions: MarketplaceOrdering
+    namespace: azure.mgmt.marketplaceordering
+    package-name: azure-mgmt-marketplaceordering
+az-output-folder: $(azure-cli-folder)/src/azure/cli/command_modules/marketplaceordering
+python-sdk-output-folder: "$(az-output-folder)/azext_[[ServiceName]]/vendored_sdks/[[ServiceName]]"
```

Comment:
the placeholder '[[ServiceName]]' should be replaced as real rp name.

Hi @yungezz ,
Was this RP released in CLI core or CLI extension? if it's in the core, line 7~16 can be removed.

Reply:
removed.